README.md

```diff
@@ -58,7 +58,7 @@ In order to deploy both the stacks the user needs to provide a set of required a
 | account (Optional)                | string  | User provided aws account                                                                                                                                                                                                                                                                        |
 | dataNodeStorage (Optional)        | string  | User provided ebs block storage size, defaults to 100Gb                                                                                                                                                                                                                                          |
 | mlNodeStorage (Optional)          | string  | User provided ebs block storage size, defaults to 100Gb                                                                                                                                                                                                                                          |
-| jvmHeapSize (Optional)            | integer | User provided JVM heap memory size (integer) in Gb, defaults to 1Gb, e.g. `--context jvmHeapSize=2` to set heap to 2Gb                                                                                                                                                                           |
+| use50PercentHeap (Optional)       | boolean | Boolean flag to use 50% of physical memory as heap.  e.g., `--context use50PercentHeap=true`                                                                                                                                                                                                     |
```

Comment:
Can you add what is the default? If it is present by default do we need to add this param just to disable it?

Reply:
The default is 1G that comes pre-configured in opensearch.yml that should not be disabled. Will add it in the README